client/src/components/ConfigTemplates/InstanceForm.vue

```diff
@@ -0,0 +1,46 @@
+<script setup lang="ts">
+import { BButton } from "bootstrap-vue";
+
+import FormCard from "@/components/Form/FormCard.vue";
+import FormDisplay from "@/components/Form/FormDisplay.vue";
+import LoadingSpan from "@/components/LoadingSpan.vue";
+
+interface Props {
+    title: string;
+    inputs: any | null; // not fully reactive so make sure these are ready to go when loading is false
```

Comment:
`any[] | null, or unknown[] | null` would be better, since inputs is of type Array.

Regarding the comment: There is no loading prop, ref or computed in this component. Is this comment outdated?

Reply:
When I wrote this component FormDisplay - which is what the inputs are sent to wouldn't react to them - so I architected it to treat the null as loading and not instantiate the FormDisplay until the inputs were set to something non-null. I think that behavior makes sense right?

Reply:
I did make it ``any[]``. I'm would like to retain the comment but I can drop it or add a redundant loading indicator if you'd like.

Reply:
How about changing it to: 
```
// not fully reactive so make sure to not mutate this array
```

You could also add a weak map key to the child component which consumes this prop. That way any time the value is reassigned, the component re-renders.
For an example of weak map keying, FormRepeat uses it if I remember correctly.

Reply:
I see what you're saying - I thought the comment was needed but the second part is definitely misleading and probably reflected an earlier version of this code. Sorry for not looking at this closer.

Reply:
I also went through and added more explicit typing throughout trying to get some obsecure typescript issue that popped up when switching ``any | null`` to ``any[] | null``. I was not able to get any version of the null handling to work so I switched null to undefined can kept the stronger typing so now it is ``Array<FormEntry> | undefined``.